.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+name: Release
+
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  build:
+    if: github.repository == 'jazzband/django-redis'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip
+          python -m pip install -U setuptools twine wheel
+
+      - name: Build package
+        run: |
+          python setup.py --version
+          python setup.py sdist --format=gztar bdist_wheel
+          # Allow twine check errors for now because our README has sections
+          # that will produce errors, but have been uploaded in the past.
+          #
+          # PR: https://github.com/pypa/readme_renderer/pull/199
+          twine check dist/* || true
```

Comment:
I just ran `twine check dist/*`and didn't see an error, so I think you can remove the `|| true` again.